Backend/src/org/tah/model/Task.java

```diff
@@ -1,31 +0,0 @@
-package org.tah.model;
-
-public class Task {
```

Comment:
you deleted the files or wanted to move them as part of reorg to the source sets?